include/envoy/buffer/buffer.h

```diff
@@ -449,6 +449,16 @@ class Instance {
    * @param watermark supplies the buffer high watermark size threshold, in bytes.
    */
   virtual void setWatermarks(uint32_t watermark) PURE;
+
+  /**
+   * Set the buffer's low and high watermark.
+   * The low_watermark should be < high_watermark if these are set. Set both
+   * watermarks to 0 to disable watermark functionality.
+   * @param low_watermark supplies the buffer low watermark size threshold, in bytes.
+   * @param high_watermark supplies the buffer high watermark size threshold, in bytes.
+   */
+  virtual void setWatermarks(uint32_t low_watermark, uint32_t high_watermark) PURE;
```

Comment:
Fairly sure we don't need this as part of the public API.  The setWatermarks 2 arg method is on the way out, would it be helpful to get it removed everywhere?  Probably as part of a separate PR.